UI/CUIControls.cpp

```diff
@@ -29,6 +30,9 @@ namespace {
     void PlayButtonCheckSound()
     { Sound::GetSound().PlaySound(GetOptionsDB().Get<std::string>("UI.sound.button-click"), true); }
 
+    void PlayButtonCheckedSound(bool b)
```

Comment:
presumably this was due to not know how to use boost::bind to connect signals to functions with different signatures (return + parameter types)